src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -376,6 +376,8 @@ def update_policy_for_item(cmd, client, resource_group_name, vault_name, item, p
 
 
 def backup_now(cmd, client, resource_group_name, vault_name, item, retain_until):
+    if retain_until is None:
+        retain_until = (datetime.utcnow() + timedelta(days=30)).strftime('%d-%m-%Y')
```

Comment:
Can you add in the help message for the default value?

Reply:
Added